addons/binding/org.openhab.binding.seneye/ESH-INF/thing/thing-types.xml

```diff
@@ -107,4 +111,28 @@
 		<description>The time your slide expires</description>
 		<state readOnly="true" />
 	</channel-type>
+	<channel-type id="wrongslide">
+		<item-type>String</item-type>
+		<label>Wrong slide</label>
```

Comment:
`Slide` should have upper case first character (like in the description here) I general as a openHAB convention words in labels should all start with upper case.